lib/techschool_web/live/course_live/index.ex

```diff
@@ -69,7 +70,8 @@ defmodule TechschoolWeb.CourseLive.Index do
     socket
     |> assign(:offset, offset)
     |> assign(:has_more_courses_to_load, has_more_courses_to_load)
-    |> stream(:courses, courses)
+    |> assign(:has_courses, !Enum.empty?(courses))
+    |> stream(:courses, courses, reset: true)
```

Comment:
Here, since we are loading more courses into the screen, I don't want to reset the other ones already loaded

```suggestion
    |> assign(:has_courses, !Enum.empty?(courses))
    |> stream(:courses, courses)
```

Reply:
I tried here, if I remove the :reset, once I get to the first select and filter something, the URL gets updated but not the records on the page. Are you sure it needs to be removed?!

Reply:
Really? I tested locally now and everything seems to be working normally 🤔
You just need to remove the `reset: true` inside the `handle_event("load_more")` and keep the one inside `handle_params`

Reply:
Pushed! d12bede5d04c01b4dd252da41ec64f7ece113bf5